setup.py

```diff
@@ -44,14 +44,17 @@
         'requests',
         'tensorflow-federated',
     ],
-    # TensorFlow doesn't yet support Python 3.9.
-    python_requires='>=3.6, <3.9',
+    # TensorFlow support Python 3.9.
+    python_requires='>=3.6
```

Comment:
I think you still need to close the string and add a comma to make this syntactically valid Python.

Reply:
Done!